irohad/ametsuchi/impl/postgres_query_executor.cpp

```diff
@@ -901,6 +901,10 @@ namespace iroha {
                           shared_model::interface::Amount(amount)));
                     });
             });
+            if (assets.empty()) {
```

Comment:
Will we cover the branch in tests somehow?

Reply:
I think we should. I will add a basic test to this PR.